make/lib/Awt2dLibraries.gmk

```diff
@@ -433,7 +433,7 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBJAVAJPEG, \
     OPTIMIZATION := HIGHEST, \
     CFLAGS := $(CFLAGS_JDKLIB), \
     HEADERS_FROM_SRC := $(LIBJPEG_HEADERS_FROM_SRC), \
-    DISABLED_WARNINGS_gcc := clobbered implicit-fallthrough shift-negative-value, \
+    DISABLED_WARNINGS_gcc := clobbered implicit-fallthrough shift-negative-value array-bounds, \
```

Comment:
Does this warning occur many times or would it make sense to disable it with pragmas in a few individual places?

Reply:
I think it is fixed upstream here:
https://github.com/openjdk/jdk11u-dev/pull/700/files

Reply:
This is a backport from upstream, I don't want to make divergent changes.

Reply:
@mrserb Thanks! A few of these may be backported by Paul already, let me rebase on nightly and see. I don't want to wait for 11u-dev to get merged into 11u.